python/sglang/srt/conversation.py

```diff
@@ -412,9 +412,17 @@ def generate_chat_conv(
                         num_image_url += 1
                         conv.modalities.append(content.modalities)
                 if num_image_url > 1:
-                    image_token = "<image>"
+                    image_token = (
+                        "<image>"
+                        if conv.name != "qwen2-vl"
+                        else "<|vision_start|><|image_pad|><|vision_end|>"
```

Comment:
can we store the image token in the chat template?

Reply:
done